src/main/java/az/gdg/mssubscriber/controller/SubscriberController.java

```diff
@@ -0,0 +1,38 @@
+package az.gdg.mssubscriber.controller;
+
+import az.gdg.mssubscriber.model.SubscriberRequest;
+import az.gdg.mssubscriber.model.dto.SubscriberDTO;
+import az.gdg.mssubscriber.service.SubscriberService;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequestMapping("/subscriber")
+@RestController
+public class SubscriberController {
+
+    private final SubscriberService subscriberService;
+
+    public SubscriberController(SubscriberService subscriberService){
+        this.subscriberService = subscriberService;
+    }
+
+    @ApiOperation("get all subscribers")
+    @GetMapping
+    public List<SubscriberDTO> getAllSubscribers(){
+        return subscriberService.getAllSubscribers();
+    }
+
+    @ApiOperation("create subscriber")
+    @PostMapping
+    public void createSubscriber(@RequestBody SubscriberRequest subscriberRequest){
```

Comment:
SubscriberDTO

Reply:
But SubscriberDTO keeps id variable that's why I can't a new subscriber

Reply:
Please read all the comments and you will understand why : 
https://github.com/GDG-Baku/ms-subscriber/pull/2#discussion_r393978094
If you read this comment, you will understand why you do not need id, therefore you do not need 2 separate classes.